docs/modules/ROOT/pages/explanations/infrastructure-as-code.adoc

```diff
@@ -1,21 +1,23 @@
-= Infrastructure as Code
+= Infrastructure as Code (IaC)
 
 Deploying Kubernetes clusters from scratch has become an easy task with IaaS interfaces.
 However, the proper management of all components that make up the cluster
 and its ecosystem can quickly become a nightmare if everything was done manually.
 
-The Infrastructure as Code approach has long proved its worth to deploy
-and maintain complex environments on the long run.
-In particular, in the context of Cloud resources, it allows to make sure
+The IaC approach has long proved its worth when deploying and maintaining complex environments 
+on the long run.
+In particular, in the context of Cloud resources, it allows sysadmins to make sure
```

Comment:
Not really. Some devs also use IaC to manage cloud resources of their own (RDS databases for ex). Some of our devs use Terraform actually.